bookkeeper-server/src/main/java/org/apache/bookkeeper/conf/ServerConfiguration.java

```diff
@@ -42,6 +42,10 @@
     protected static final String MINOR_COMPACTION_THRESHOLD = "minorCompactionThreshold";
     protected static final String MAJOR_COMPACTION_INTERVAL = "majorCompactionInterval";
     protected static final String MAJOR_COMPACTION_THRESHOLD = "majorCompactionThreshold";
+    protected static final String MAJOR_MEDIAN_COMPACTION_THRESHOLD = "majorMedianCompactionThreshold";
```

Comment:
add these new settings to https://github.com/apache/bookkeeper/blob/master/bookkeeper-server/conf/bk_server.conf and https://github.com/apache/bookkeeper/blob/master/site/_data/config/bk_server.yaml